setup.py

```diff
@@ -42,6 +42,7 @@
     'flake8',
     'flake8-commas',
     'flake8-quotes',
+    'flake8-spellcheck',
```

Comment:
To fix py2 we could just turn this into a list and append this requirement only if py3.

Reply:
Actually - better than that - we can just modify this line (discovered via https://stackoverflow.com/questions/34407802/how-can-i-specify-a-requirement-only-for-python-2-ie-not-python-3-in-requireme):

```py
    'flake8-spellcheck ; python_version >= "3"',
```

Reply:
Realised I could just do it as such a small change; hope you don't mind! https://github.com/Fizzadar/pyinfra/pull/217/commits/fb96848867e269c36e220a98d8717f444486f71a

Reply:
No worries!

Thanks